tests/test_routing.py

```diff
@@ -43,3 +45,23 @@ def test_request_with_trailing_slash_not_redirected(
     """
     response = test_app.get(invalid_route)
     assert response.status_code == 404
+
+
+@pytest.mark.parametrize(
+    "test_root_path,expected_status_code",
```

Comment:
```suggestion
    "test_route,expected_status_code",
```
Just like above I think this is a better name because in the test you keep the `root_path` of the app constant, but you change the `route` (prefix) you're sending GETs to